install/supabase-install.sh

```
@@ -0,0 +1,155 @@
+msg_info "Installing Docker"
+$STD apt-get install -y \
+  docker-ce \
+  docker-ce-cli \
+  containerd.io \
+  docker-buildx-plugin \
+  docker-compose-plugin
+msg_ok "Installed Docker"
+
+msg_info "Downloading Supabase"
+mkdir -p /opt/supabase
+cd /opt/supabase
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)